irs-api/src/main/java/org/eclipse/tractusx/irs/ess/service/EdcRegistration.java

```diff
@@ -208,6 +208,7 @@ private HttpEntity<String> toEntity(final String body) {
         return new HttpEntity<>(body, headers());
     }
 
+    @SuppressWarnings({ "PMD.LooseCoupling" })
```

Comment:
Please add a description on why this is suppressed. e.g. "Intended use of implementation HttpHeaders instead of MultiValueMap"